code/script.js

```diff
@@ -227,99 +229,146 @@ const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
+
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
+  winOrLoseWrapper.style.display = "none";
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard();
+  setSecret();
+  console.log(secret);
```

Comment:
According to Technigo's teacher ,console.logs should be removed before pushing the projects as it's just for developers to see behind the scenes.